server/src/resolvers/mutations/createOrUpdateDate.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    createOrUpdateDate: async (_, { dateId, dateObjectId, childId, month, day, year, hours, dayOfWeek, notes, paid, isFixedRate }, context) => {
```

Comment:
Maybe we should pull this in as arg then de-structure inside of the fun body. Also the create and update are the same it looks like so we can clean that up too.